internal/commands/app/create_inputs_test.go

```diff
@@ -167,13 +167,43 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Nil(t, err)
 		assert.Nil(t, testFile.Close())
 
-		dir, err := inputs.resolveDirectory(profile.WorkingDirectory)
+		dir, err := inputs.resolveDirectory(nil, profile.WorkingDirectory)
 
 		assert.Nil(t, err)
 		assert.Equal(t, path.Join(profile.WorkingDirectory, appName), dir)
 		assert.Nil(t, os.Remove(appName))
 	})
 
+	t.Run("should return path of wd with a new app name appended trying to write to a local directory", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
+		defer teardown()
+
+		_, console, _, ui, err := mock.NewVT10XConsole()
+		assert.Nil(t, err)
+		defer console.Close()
+
+		doneCh := make(chan (struct{}))
+		go func() {
+			defer close(doneCh)
+
+			console.ExpectString("Directory './test-app' already exists, writing app contents to that destination may result in file conflicts.")
+			console.ExpectString("Would you still like to write app contents to './test-app'? ('No' will prompt you to provide another destination)")
+			console.SendLine("no")
```

Comment:
[curious] does typing full "yes" actually trigger an affirmative confirm here?  i thought it was single characters before (y/n)...

Reply:
It supports all capitalization variations of 'y', 'yes', 'n', and 'no'.